docs/documentation/features/scenes.md

```diff
@@ -0,0 +1,113 @@
+---
+layout: documentation
+---
+
+{% include base.html %}
+
+# Scenes
+
+In the general context of home automation, a scene is a defined set of states of one or more home devices. For example, a **night scene**, which turns on the indoor lights after sunset. The notion of scenes is not directly available in Eclipse Smarthome but scenes can be created by the means of using rules.
+
+## Concept
+
+In Eclipse Smarthome, a scene is created by defining states for different items inside action section of a rule. The triggers and conditions of the rule are left empty. Once the rule is activated, the actions set the state of the items thereby activating a specific scene.
+
+The scene can only be activated and no deactivation is possible. It can be activated either manually or automatically. To manually activate the scene you will need to make a call to rest endpoint `PUT /rest/rules/{ruleUID}/runnow`. To automate the scene activation, a second rule must be created. The action section of this second rule is responsible for executing the scene with the `core.RunRuleAction` actionhandler. 
+
+The action module, used for running the scene, must support the direct execution of rules without requiring to evaluate the conditions. This way all actions will be executed even if an action before returns an error.
+
+Asynchronous execution of actions is needed to fulfil further requirements:
+
+* Most of the actions just send events. And as the eventHandlers for these events are executed asynchronously the resulting action execution is also asynchronous.
+* For long-running script actions, e.g., the handler should be aware of asynchronous execution.
+
+
+## Defining Scene via JSON
```

Comment:
This subsection contains not only JSON examples for defining scenes, but also for activating scenes. Hence, I suggest using another title for this subsection (like, e.g., "Scene creation and activation using the REST API")